lib/device.js

```diff
@@ -142,8 +156,22 @@ class Device extends EventEmitter {
 				if (err instanceof particleUsb.TimeoutError && Date.now() >= timeoutAt) {
 					// "Request timeout" -> "Test timeout"
 					throw new Error('Test timeout');
+				} else if (err instanceof RunnerError) {
+					if (expectingSafeMode && err.code === RequestResult.NOT_SUPPORTED && Date.now() < timeoutAt) {
```

Comment:
I'm guessing there isnt any adverse side effects of polling a device in safe mode with the USB control requests?